__tests__/topics.test.js

```diff
@@ -0,0 +1,28 @@
+const app = require('../app.js');
+const request = require('supertest');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js')
+const topicData = require('../db/data/test-data/index.js');
+
+beforeEach(() => seed(topicData));
+afterAll(() => db.end());
+
+describe('api/topics', () => {
+    test('GET: 200, an array of topic objects with properties "slug" and "description"', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.topics).toEqual(expect.any(Array));
+            expect.arrayContaining(['slug', 'description']);
```

Comment:
I'd like you to revisit this test. We should have an assertion for the length of the array we are expecting. As the topics are an array, what method could you use to iterate over that array and check the object properties? If you use an array method, you do not need to assert that the database is returning any array.